lib/client-assets.php

```diff
@@ -166,18 +167,69 @@ function gutenberg_override_translation_file( $file, $handle ) {
  *                                 Default 'all'. Accepts media types like 'all', 'print' and 'screen', or media queries like
  *                                 '(orientation: portrait)' and '(max-width: 640px)'.
  */
-function gutenberg_override_style( $handle, $src, $deps = array(), $ver = false, $media = 'all' ) {
-	wp_deregister_style( $handle );
-	wp_register_style( $handle, $src, $deps, $ver, $media );
+function gutenberg_override_style( &$styles, $handle, $src, $deps = array(), $ver = false, $media = 'all' ) {
+	$style = $styles->query( $handle, 'registered' );
+	if ( $style ) {
+		$styles->remove( $handle );
+	}
+	$styles->add( $handle, $src, $deps, $ver, $media );
 }
 
+/**
+ * Registers vendor JavaScript files to be used as dependencies of the editor
+ * and plugins.
+ *
+ * This function is called from a script during the plugin build process, so it
+ * should not call any WordPress PHP functions.
```

Comment:
Does this mean the function belongs in a separate file? Possibly something to clean up after this PR.

Reply:
I moved the definition up and didn't touch the description. I honestly don't know 🤷‍♂

Reply:
However, this made me think that we use it somewhere else 😅  I will double-check.

Reply:
I found the usage:

https://github.com/WordPress/gutenberg/blob/f8b62bd29fc83972256d7b2315dee2341af06cc1/bin/build-plugin-zip.sh#L77-L79

I will test on Monday before merging.

Reply:
Fixed in 3f3d015. Tested with `npm run plugin-build`:

<img width="678" alt="Screen Shot 2019-10-28 at 10 59 19" src="https://user-images.githubusercontent.com/699132/67669485-48887080-f972-11e9-96e1-4efd87242dd1.png">